js/app.js

```diff
@@ -74,8 +83,9 @@ $(document).ready(function() {
     $('form')[0].reset();
   });
 
-
-  $(function() {
+  // If you can tell me how this function below works then you won't need to provide a link
+  // to where you saw it used.
```

Comment:
http://paulmason.name/item/simple-jquery-carousel-slider-tutorial
This is the link i followed